e2e/listViewSpec.js

```diff
@@ -7,6 +7,9 @@ testMethod('ListView', function () {
     'use strict';
 
     beforeEach(function () {
+        // Reload page to init FakeRest data
+        browser.get(browser.baseUrl);
```

Comment:
Can't you do that in the `onPrepare` protractor configuration file ? So, this will works for all tests.